modules/ROOT/pages/getting.started.lake.adoc

```diff
@@ -0,0 +1,229 @@
+= Getting Started with VantageCloud Lake 
+:experimental:
+:page-author: Vidhan Bhonsle
+:page-email: vidhan.bhonsle@teradata.com
+:page-revdate: January 2nd, 2024
+:description: Create your own environment in VantageCloud Lake
+:keywords: data warehouses, compute storage separation, teradata, vantage, cloud data platform, business intelligence, enterprise analytics, jupyter, teradatasql, ipython-sql, teradatasqlalchemy, vantagecloud, vantagecloud lake, ip address, public internet, lake
+
+== Overview
+
+Teradata VantageCloud Lake is Teradata's next-generation, cloud-native analytics and data platform. It provides lakehouse deployment patterns along with the ability to run independent elastic workloads using an object store-centric design. +
```

Comment:
We should say `object storage` instead of `object store`. That's the industry standard way of referring to s3-like buckets.